traffic_ops/traffic_ops_golang/monitoring/monitoring_test.go

```diff
@@ -454,7 +454,7 @@ func TestGetConfig(t *testing.T) {
 		t.Fatalf("creating transaction: %v", err)
 	}
 
-	sqlConfig, err := getConfig(tx)
+	sqlConfig, err := getConfig(tx, "")
```

Comment:
Instead of using an empty string for the CDN name, the mock query should specify a CDN name as an argument.

Reply:
Done